frontend/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,5 @@
+import Image from "next/image"
```

Comment:
We already have a Logo component at `shared-components/logo.tsx` (& it is/should be sized 144x113 as per wireframes)